app/Filament/Resources/InterpretationResource.php

```diff
@@ -48,16 +48,54 @@ public static function table(Table $table): Table
     {
         return $table
             ->columns([
-                Tables\Columns\TextColumn::make('name'),
-                Tables\Columns\TextColumn::make('namespace'),
-                Tables\Columns\TextColumn::make('route'),
-                Tables\Columns\TextColumn::make('updated_at')->dateTime(),
+                Tables\Columns\TextColumn::make('name')->disableClick(),
+                Tables\Columns\TextColumn::make('context')
+                    ->label(__('Show context'))
+                    ->getStateUsing(fn (Interpretation $record): string => __('Show context').' <span class="sr-only"> '.__('for').$record->name.'</span>')
```

Comment:
This is still missing a space between "for" and the name, so it comes out like "forWhat is the Accessibility Exchange?"